pages/md-style-guide.md

```diff
@@ -143,3 +143,40 @@ Danger is a special type of info box in which text showing any danger can be sho
 #### Example
 
 <Danger label="label"> message to show. </Danger>
+
+---
+
+### **Table of Contents**
+
+Table of Contents provides a structured overview of the main sections or chapters within the current page/document.
+
+```markdown
+<TableOfContent content={content} depth={depth} />
+```
+
+#### Example
+
+<TableOfContent depth={5} />
+
+---
+
+### **Tabs Group**
+
+This can be used to group multiple tabs under a common name and differentiated by specific labels and content.
+
+```markdown
+[tabs-start "label"]
+[tab "name"]
+message to show
+```
```

Comment:
When I add this line:
```markdown
[tabs-end]
```
 This gets treated as a tab group and gets rendered rather than displaying as a code block. Any help on this appreciated.

Reply:
Adding one newline [enter] might help in this

Reply:
@DhairyaMajmudar Is this what you meant?
![image](https://github.com/json-schema-org/website/assets/81156510/24a36abf-6ab1-4831-8961-fd73ddecedb1)

If so it is still the same.
![image](https://github.com/json-schema-org/website/assets/81156510/39e45a00-1a86-45ac-bcf6-fb3d23be0c0a)

Reply:
Yeah, add newline before [tab-end]

Reply:
Still the same😢

Reply:
Added newline before, after and both. All three yeild the same result as above.

Reply:
let me check if I can find any another solution 👍🏻